addons/l10n_es_edi_tbai/models/account_edi_format.py

```diff
@@ -111,6 +111,8 @@ def _l10n_es_tbai_post_invoice_edi(self, invoice):
             chain_head = invoice.company_id._get_l10n_es_tbai_last_posted_invoice()
             if chain_head and chain_head != invoice and not chain_head._l10n_es_tbai_is_in_chain():
                 raise UserError(f"TicketBAI: Cannot post invoice while chain head ({chain_head.name}) has not been posted")
+            if invoice.move_type == 'out_refund' and not invoice.reversed_entry_id._l10n_es_tbai_is_in_chain():
+                raise UserError(f"TicketBAI: Cannot post a reversal move while the source document ({invoice.reversed_entry_id.name}) has not been posted")
```

Comment:
It's only missing the translation (i know the error above didn't have the translation, maybe we can add it also), also by using _() no need for the fstring i think :smile:

Reply:
Indeed, it's a good idea, thanks :)

Reply:
Maybe i should have been more precise on that but unfortunately the pot is not really sufficient on his own for localisation, can you export the po also and add the spanish translation (you can use a translator like deepl it will be ok :smile:)

Reply:
My bad, I always forget what we have/don't have on transifex

Reply:
The rules of thumb for localisation is that we always need a pot and po files :smile: